src/components/status/godkjennplansendt/Infobox.tsx

```diff
@@ -1,42 +1,48 @@
 import { Alert, BodyShort, Heading } from "@navikt/ds-react";
-import styled from "styled-components";
-
-const AlertStyled = styled(Alert)`
```

Comment:
Hadde det vært bedre å endre margin for AlertStyled isteden for å legge til en ny div?

Reply:
Tja ja kanskje, eneste er om vi vil ha 1 sted å tweake margin-bottom på. Har brukt den SpacedDiv greia veldig mange steder for å få lik margin-bottom. Men ikke sikkert det er så viktig